bundle-generator/deployment/src/test/java/io/quarkiverse/operatorsdk/bundle/MultipleOperatorsBundleTest.java

```diff
@@ -49,6 +48,13 @@ public void shouldWriteBundleForTheOperators() throws IOException {
         assertEquals(FirstReconciler.REPLACES, csv.getSpec().getReplaces());
         var bundleMeta = getAnnotationsFor(bundle, "first-operator");
         assertEquals(BUNDLE_PACKAGE, bundleMeta.getAnnotations().get("operators.operatorframework.io.bundle.package.v1"));
+        assertEquals(2, csv.getSpec().getInstallModes().size());
+        var installMode = csv.getSpec().getInstallModes().get(0);
+        assertEquals("AllNamespaces", installMode.getType());
```

Comment:
I'm always debating whether or not to use constants in the tests, especially when the tests are checking compatibility with an external service or spec. In this case, if we used a constant, and the constant value changed **but not the external spec**, then the test would still pass even though it would now output incorrect values.